doc/application/index.rst

```diff
@@ -299,19 +294,34 @@ The Zephyr build system compiles and links all components of an application
 into a single application image that can be run on simulated hardware or real
 hardware.
 
-As described in :ref:`getting_started_cmake`, on Linux and macOS you can choose
-between the ``make`` and ``ninja`` generators, whereas on Windows you need to use
-``ninja``. For simplicity we will use ``ninja`` throughout this guide.
+Like with any other CMake-based system, the build process takes place :ref:`in
+two stages <cmake-details>`. First build files (also known as buildsystem) are
+generated using the ``cmake`` command-line tool while specifying a generator.
+This generator determines which native build tool to generate a buildsystem for.
+The second stage consists on running the native build tool to actually build the
```

Comment:
consists of

Reply:
Changed to @dbkinder's suggestion instead